src/core/config/platform.h

```diff
@@ -166,14 +166,4 @@
 #endif
 #endif
 
-/**
- * @def OPENTHREAD_CONFIG_PLATFORM_CSL_UNCERT
- *
- * The Uncertainty of the scheduling CSL of transmission by the parent, in ±10 us units.
- *
- */
-#ifndef OPENTHREAD_CONFIG_PLATFORM_CSL_UNCERT
-#define OPENTHREAD_CONFIG_PLATFORM_CSL_UNCERT UINT8_MAX
```

Comment:
Why is this config removed?

If we want to remove it we need to make sure if any project/build that used this noticed this removal.
For this, we add a check in `openthread-core-config-check` to give error if it is being used.

Reply:
I'm fine with leaving it, but found difficult to make use of this config for Posix platform. Anyway platforms should implement the weak `otPlatRadioGetCslClockUncertainty` and eventually this function could return some variable result depending on the device conditions.